riscv/src/continuations.rs

```diff
@@ -152,10 +186,24 @@ pub fn rust_continuations_dry_run<F: FieldElement>(
     let program = pipeline.compute_analyzed_asm().unwrap().clone();
     sanity_check(&program);
 
+    log::info!("Initializing memory merkle tree...");
+
+    // Get initial memory contents from the special variable "initial_memory".
+    // In the first full run, we use it as the memory contents of the executor;
+    // on the independent chunk runs, the executor uses zeroed initial memory,
+    // and the pages are loaded via the bootloader.
```

Comment:
So the main difference is that now the bootloader takes the [potentially] non-zero initial memory into account when computing the values of the used pages for the first chunks? The bootloader still loads the used memory pages for the first chunk, right?

Reply:
> The bootloader still loads the used memory pages for the first chunk, right?
The bootloader wasn't loading anything in the first chunk. Now it is.